datacommons_client/utils/response.py

```diff
@@ -0,0 +1,311 @@
+from dataclasses import asdict, dataclass, field
```

Comment:
What do you think about moving this file to the `datacommons_client/endpoints/` folder?

Reply:
That probably makes the most sense indeed @dwnoble. Further down the stack, `response.py` contains specific Response classes for each of the endpoints. So keeping them together sounds good. I'll make the change.

Reply:
File move: https://github.com/datacommonsorg/api-python/pull/204/commits/f0b7757f03ce23e832263788e02f80d954117da3